java/reflection/BaseType.java

```diff
@@ -0,0 +1,29 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Yes this is the plan not time to do it we would like to be sure first that what we did on the C++ generation part was not stupid. Thanks for your review and sorry for not respecting the Google C++ code style.